wire-library/wire-tests/src/commonTest/proto/java/custom_options.proto

```diff
@@ -30,8 +30,10 @@ message FooBar {
   optional uint64 qux = 4 [(squareup.protos.custom_options.my_field_option_one) = 18,
                            (squareup.protos.custom_options.my_field_option_two) = 34.5,
                            (squareup.protos.custom_options.my_field_option_five) = 3,
-                           (squareup.protos.custom_options.my_field_option_six) = ["a", "b"],
-                           (squareup.protos.custom_options.my_field_option_seven) = [BAV, BAX]
+                           (squareup.protos.custom_options.my_field_option_six) = "a",
+                           (squareup.protos.custom_options.my_field_option_six) = "b",
+                           (squareup.protos.custom_options.my_field_option_seven) = BAV,
+                           (squareup.protos.custom_options.my_field_option_seven) = BAX
```

Comment:
Without this change (and the similar two changes below), `protoc` complains of syntax errors:
```
custom_options.proto:33:83: Expected option value.
custom_options.proto:60:48: Expected option value.
custom_options.proto:142:38: Expected option value.
custom_options.proto:143:37: Expected option value.
```
The `protoc` parser does not really understand message literals or list literals. This list literal syntax is only supported _inside_ a message literal. `protoc` treats everything between the `{` and `}` of a message literal as an opaque blob which is later unmarshalled via the proto text format, when interpreting options.

Reply:
Ooooh I had no idea how it worked, and I couldn't find a good spec. This is rad!